install_files/ansible-base/roles/restrict-direct-access/templates/ths_config_v3.j2

```diff
@@ -0,0 +1 @@
+{% if item.item.filename.endswith('-aths') %}HidServAuth {% endif %}{{ item.stdout }}
```

Comment:
AFAICT the HidServAuth directive isn't used for v3 auth services. It might be better to create the client-side auth files here instead. So the source interface `app-source-v3ths` file would look like 
```
<onionUrl>
```
and the three `app-journalist-v3aths, app-ssh-v3aths, mon-ssh-v3aths` file would look like
```
<onionURLFirst56>:<(journalistv3|sshv3|monv3)>:x25519:<privateKey> 
```
This would a) simplify the `tailsconfig` logic and b) make the staging workflow for getting at the authenticated services roughly similar to the existing one for v2. Right now devs need to append the aths contents to a torrc (browser or daemon), for v3 they would have to copy the v3aths files to their ClientOnionAuthDir.

Reply:
The journalist keys will then have to copy more files, that is the reason I wanted to separate it out totally.

Reply:
Nope, I think all the journalist workstation would need would be the `app-journalist-v3aths` file, which `tailsconfig` would copy into place and parse for the onion URL for desktop shortcuts, which is basically what's happening already (except the `HidServAuth` line is appended to `torrc` rather than being copied as a file somewhere).

Reply:
Setting up a new Admin key _would_ require that the JSON file be copied over as well, along with `site-specific` and all the `*ths` files, but I think that we should be recommending to people that they make new Admin keys by cloning the PV from a backup and rotating the new key's SSH creds anyway.